IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -377,6 +377,22 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         return;
     }
     
+    // Handle anchor links that were clicked
+    if([navigationAction navigationType] == WKNavigationTypeLinkActivated) {
+#if TARGET_OS_IPHONE
+        UIApplication *application = [UIApplication performSelector:@selector(sharedApplication)];
+#else
+        NSWorkspace *application = [NSWorkspace sharedWorkspace];
+#endif
+        //Open secure web links with target=new window in default browser
+        if(([requestURL.scheme.lowercaseString isEqualToString:@"https"] && !navigationAction.targetFrame.isMainFrame) ||
```

Comment:
What kind of url will be here? Just curious is there will be any security concern.